.github/workflows/check_build.yml

```diff
@@ -76,6 +76,8 @@ jobs:
         with:
           path: '~/.konan/**'
           key: konan-ios
+      - name: Xcode cache
+        uses: mikehardy/buildcache-action@v1
```

Comment:
Might be worth playing with some of the [settings here](https://github.com/mikehardy/buildcache-action). Maybe if we change the cache location it’ll be reset?